### PR TITLE
Add fire-tick-delay option

### LINKED ISSUE #7277: Fire tick speed
### Is your feature request related to a problem?

Griefing

### Describe the solution you'd like.

A config option to set fire tick speed.

### Describe alternatives you've considered.

Looking for another plugin without success.

### Other

I noticed fires on my server spread very quickly and would like to slow them down. I however do **not** want to disable fire spread entirely.

### PR DESCRIPTION
**Pull request by vytskalt**
Closes #7277

### CODE REVIEW

File: patches/server/0918-Add-fire-tick-delay-option.patch [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+diff --git a/src/main/java/io/papermc/paper/configuration/WorldConfiguration.java b/src/main/java/io/papermc/paper/configuration/WorldConfiguration.java
+index 32f0cd29d1198fe320d10ccfe0b02f8632ac12aa..4b5372487e9a905d874da9c095351318988e118b 100644
+--- a/src/main/java/io/papermc/paper/configuration/WorldConfiguration.java
++++ b/src/main/java/io/papermc/paper/configuration/WorldConfiguration.java
+@@ -334,6 +334,7 @@ public class WorldConfiguration extends ConfigurationPart {
+             public BooleanOrDefault findAlreadyDiscoveredLootTable = BooleanOrDefault.USE_DEFAULT;
+         }
+ 
++        public int fireTickDelay = 30;
```
Comment (electronicboy) [inline patches/server/0918-Add-fire-tick-delay-option.patch]: This belongs in the config files patch

**Review by electronicboy — CHANGES_REQUESTED** [review]

**Review by electronicboy — APPROVED** [review]

**Review by Owen1212055 — APPROVED** [review]
To be honest, it would probably be less of a diff if you just modify the pre-existing method and add a level param. 

But looks good here.

**Comment by vytskalt** [comment]
> To be honest, it would probably be less of a diff if you just modify the pre-existing method and add a level param.

Done

**Comment by ByteBeamer** [comment]
Thanks so much for this!

**Pull request merged**